src/cl_main.cpp

```diff
@@ -6662,6 +6662,124 @@ void ServerCommands::SetLineTextureByID::Execute()
 		CLIENT_PrintWarning( "SetLineTextureByID: unknown texture: %s\n", textureName.GetChars() );
 }
 
+//*****************************************************************************
+//
+void ServerCommands::SetLineTextureOffset::Execute()
+{
+	if ( line->sidedef[side] == NULL )
+		return;
+
+	line->sidedef[side]->textures[position].xoffset = XOffset;
+	line->sidedef[side]->textures[position].yoffset = YOffset;
+}
+
+//*****************************************************************************
+//
+void ServerCommands::SetLineTextureOffsetByID::Execute()
+{
+	int linenum = -1;
+	const fixed_t NO_CHANGE = 32767 << FRACBITS;
```

Comment:
This code here is reimplmenting `LS_Line_SetTextureOffset`, right? You should be able to directly call `LS_Line_SetTextureOffset` instead of reimplementing it here. The line special functions are accessible via the `LineSpecials` array exported in `p_lnspec.h`.

Reply:
Sounds good, thanks for the suggestion.